### DISCUSSION TITLE
Project actions cause too much noise on issue timelines

### DISCUSSION THREAD

**Discussion by jeffhandley** [community/community#12335] [Projects and Issues]
As we use GitHub projects more and more, we're finding that the amount of noise generated on the issue pages is overwhelming. Every time an issue is added to a project, moves from one column to another, or gets removed from a project, an event is shown on the timeline on the issue's page. The amount of noise generated is affecting our eagerness to use project boards.

I could imagine a setting or a toggle for choosing whether to show these project activities. That could even be generalized to allow the selection of which activities should be shown--perhaps changes to labels, milestones, or other actions could be unchecked for improved usability in some scenarios too.

Alternatively, I could imagine a project board to be configured to indicate whether or not the activities should be reflected on the issue page. For our private project boards that we use for issue triage, we'd choose to _never_ show that activity on the pages, as the history is actually useless for us.

**Comment by mfsbo** [comment]
This seems to be asking to control the amount of activity on timeline and https://github.com/community/community/discussions/16642 seems to be asking to include. I think both are same issues and control to allow amount of activity on time line based on Labels, Custom Fields and Swimlanes would be great.

**Comment by tomparkp** [comment]
I wanted to second this and add a clear example. I'm become more and more iterative throughout my career (as tends to be encouraged in agile teams) and love working with my team to continuously refine our processes. As a result I'm often testing different approaches, naming conventions, etc. to get a feel for what works best.

The result is an extremely overwhelming feed of information within our issues:
![2023-01-17-18 08 45](https://user-images.githubusercontent.com/2266762/213068492-a11ba1eb-65f7-4ea4-a497-5d0a4617858b.png)

Its extreme to the point that it drowns out the important aspect of issues, which is conversations. I think a (relatively) simple solution would be to collapse large amounts of history updates by default and allow users to expand them to allow focusing on conversations. A more robust solution be something like @jeffhandley is describing which is allowing customization of whats visible in the history of issues.
